### PR TITLE
Add Polywork Icon

### LINKED ISSUE #1006: [FEATURE] Add Polywork as Link
### Description

I saw popular supported buttons but it would be great if we add the polywork.com link to the linkfree.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by parisioann**
## Fixes Issue

Closes #1006 

## Changes proposed

Polywork Icon has been added, as requested by a user. Now by including -> "icon": "polywork" and "url": "<url to your Polywork profile>", users will be able to connect their Polywork profile link.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

polywork

## Note to reviewers

This is my first contribution/PR, some feedback would be greatly appreciated. Thanks 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1021"></a>

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Thanks for submitting a PR! 😄 You need to add the Polywork icon at [`src/config/links.json`](https://github.com/EddieHubCommunity/LinkFree/blob/main/src/config/links.json). Let us know if you need any help with that! 😃

**Comment by emmalearnscode** [comment]
> Hey there! Thanks for submitting a PR! 😄 You need to add the Polywork icon at [`src/config/links.json`](https://github.com/EddieHubCommunity/LinkFree/blob/main/src/config/links.json). Let us know if you need any help with that! 😃

And also to the readme 😊

**Comment by parisioann** [comment]
> > Hey there! Thanks for submitting a PR! 😄 You need to add the Polywork icon at [`src/config/links.json`](https://github.com/EddieHubCommunity/LinkFree/blob/main/src/config/links.json). Let us know if you need any help with that! 😃
> 
> And also to the readme 😊

@emmalearnscode Hey thanks for this :) Just some help if you don't mind. I can see (in the README.md file) each icon gets the actual icon/img from a link e.g. https://user-images.githubusercontent.com/91655303/148160774-755adc38-e089-4a20-910f-292b890e2c63.png. How can I do that for the Polywork icon? Thanks

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.59.1",
+      "version": "0.60.2",
```
Comment (Panquesito7) [inline package-lock.json:9]:
```suggestion
      "version": "0.59.1",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by emmalearnscode** [comment]
I see that you closed this pull request - Did you need some help with it@parisioann ? It was almost ready for merging.

**Comment by parisioann** [comment]
Hey @emmalearnscode just messed up while was trying to merge. There were new changes/conflicts and did not know how to proceed. I will try again thnx

**Pull request closed without merging**